cognite/neat/_rules/models/information/_rules_input.py

```diff
@@ -36,8 +36,8 @@ class InformationInputMetadata(InputComponent[InformationMetadata]):
     description: str | None = None
     created: datetime | str | None = None
     updated: datetime | str | None = None
-    license: str | None = None
-    rights: str | None = None
+    physical: str | None = None
+    conceptual: str | None = None
```

Comment:
Is the thinking here to use these two fields to link the different input rules?

Reply:
Correct !